File: TheQueue.Server.Core/Models/BroadcastMessages/SupervisorStatus.cs
using TheQueue.Server.Core.Enums;

namespace TheQueue.Server.Core.Models.BroadcastMessages
{
	public class SupervisorStatus
	{
        public string Name { get; set; }
        public Status Status { get; set; }
        public QueueStatus? Client { get; set; }
    }
}

File: TheQueue.Client.Common/src/main/java/Models/BroadcastMessages/SupervisorStatus.java
package Models.BroadcastMessages;

import com.fasterxml.jackson.annotation.JsonProperty;
import Enums.Status;

public class SupervisorStatus {
    @JsonProperty("name")
    public String Name;
    @JsonProperty("status")
    public Status Status ;
    @JsonProperty("client")
    public QueueTicket Client;
}
